docs/sep-proj-user-doc.md

```diff
@@ -0,0 +1,78 @@
+# External portals and user documentation
+
+As of the release that includes this README, the PDP will include some
+portals that are maintained as separate projects/repos (e.g., the 
```

Comment:
Does this section get included in the user docs or is it just for devs? If it is only for devs, then just ignore the following...

I don't know that I agree with the way that this is communicated. I think that when we talk about a "portal" it's from the user/stakeholder perspective, whereas we talk about software/repositories from our own perspective as the service provider. This documentation is supposed to be user documentation, so I don't think that it's useful or even desirable to describe the nuances of what parts of the software are in which repositories or to differentiate between "external"/"internal". That's for us to worry about. The user just wants to know how to use it.

Reply:
It's just for devs. Confusingly, we have 
- the `docs` directory for what amount to README extensions, for devs
- the `doc` directory containing user documentation

I hesitate to refactor this for fear of confusing Sphinx.